sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Customizations/LiveMetricsRestAPIsForClientSDKsRestClient.cs

```diff
@@ -59,7 +59,7 @@ public QuickPulseResponse CustomPing(string ikey, string apikey = null, int? xMs
                 case 200:
                     {
                         CollectionConfigurationInfo value = default;
-                        if (message.Response.Headers.ContentLength != 0)
+                        if (message.Response.Headers.ContentLength is not null && message.Response.Headers.ContentLength != 0)
```

Comment:
Was it a bug that we are solving here?

Reply:
a bug exposed by unit tests. unknown if this would ever occur in production.
`ContentLength` is nullable and was throwing null exceptions because the MockResponse was empty.

I wrote this entire block because the generated client doesn't perform any checks, which did cause runtime bugs.